website/src/routes/[slug].svelte

```diff
@@ -0,0 +1,85 @@
+<script>
+	import { goto, stores } from "@sapper/app";
+	import { derived } from "svelte/store";
+	import { categories } from "../stores";
+
+	let photoViewerActive = false;
+	let activePhoto = "";
+	
+	// Object containing whole site data
+	let site = null;
+	let error = null;
+
+	const refreshData = derived([stores().page, categories], ([page, category]) => {
+		if (!page || !category) {
+			return false;
+		}
+
+		return category.find(it => it.name.toLowerCase() === page.params.slug);
+	});
+
+	refreshData.subscribe(siteData => {
```

Comment:
I think it would be better if we redirect to a 404 page rather then showing the error here. Additionally, should we be treating `false` and `undefined` differently? If `siteData` is `false` that can mean the categories haven't loaded yet, where as if it's `undefined` a category wasn't found.